system/Autoloader/FileLocator.php

```diff
@@ -123,7 +125,7 @@ public function getClassname(string $file): string
                 continue;
             }
 
-            if ((isset($tokens[$i - 2][1]) && ($tokens[$i - 2][1] === 'phpnamespace' || $tokens[$i - 2][1] === 'namespace')) || ($dlm && $tokens[$i - 1][0] === T_NS_SEPARATOR && $token[0] === T_STRING)) {
+            if ((isset($tokens[$i - 2][1]) && ($tokens[$i - 2][1] === 'phpnamespace' || $tokens[$i - 2][1] === 'namespace')) || ($dlm && $tokens[$i - 1][0] === T_NS_SEPARATOR && $token->is(T_STRING))) {
```

Comment:
@samsonasik 
This code causes an error.

> Running PHPStan...
Note: Using configuration file .../CodeIgniter4/phpstan.neon.dist.
Error thrown in .../CodeIgniter4/system/Autoloader/FileLocator.php on line 128 while loading bootstrap file .../CodeIgniter4/phpstan-bootstrap.php: Cannot use object of type PhpToken as array

Reply:
Possibly bug on TokenGetAllToObjectRector

Reply:
Yes, this is a bug on TokenGetAllToObjectRector. 
The `$tokens` in the original code is just an array of array, and used as `$tokens[$i - 2][1]`.
But `$tokens` in the refactored code is an array of PhpToken object.

Reply:
Skipped the rule.